File: exercicios-cpp/estrutura-sequencial/exe12.cpp
#include <iostream>
#include <locale>
#include <iomanip>

/*
Tendo como dados de entrada a altura de uma pessoa, construa um algoritmo
que calcule seu peso ideal, usando a seguinte fórmula: (72.7*altura) - 58
*/

using namespace std;

int main(){
	setlocale(LC_ALL, "portuguese");
	
	float altura=0, peso_ideal=0;
	
	cout << "Informe a sua altura: "; cin >> altura;
	peso_ideal = (72.7 * altura) - 58;
	
	cout << fixed << setprecision(2);
	
	cout << "\nCom relação a sua altura, seu peso idaal seria " << peso_ideal << "Kg";
	
	
	return 0;	
}

File: exercicios-javascript/estrutura-sequencial/exe12.js

/*
Tendo como dados de entrada a altura de uma pessoa, construa um algoritmo que calcule seu peso ideal, usando a seguinte fórmula: (72.7*altura) - 58
*/

const readline = require('readline-sync')

let altura = readline.questionFloat('\nInforme a sua altura: ')

const peso_ideal = (72.7 * altura) - 58

console.log(`Em relacao a sua altura seu peso ideal seria ${peso_ideal} Kg`);
